lib/modules/hide_downvote.js

```diff
@@ -0,0 +1,13 @@
+HNSpecial.settings.registerModule("hide_downvote", function () {
+  function hideDownvoteButton () {
+    var arrows = _.toArray(document.getElementsByClassName('votearrow'));
+
+    arrows.forEach(function(arrow) {
+      if (arrow.classList.contains('rotate180')) {
+        arrow.parentElement.remove();
+      }
+    });
+  }
+
+  hideDownvoteButton();
```

Comment:
Can't remember if that's ever the case: I don't think there's "load more" functionality on comments. So I think for now it shouldn't be necessary :)